test/unit/tessel.js

```diff
@@ -48,9 +54,7 @@ exports['Tessel (get)'] = {
 
   infoOutput: function(test) {
     test.expect(1);
-    Tessel.get({
-        timeout: 0.01
-      })
+    Tessel.get(this.standardOpts)
```

Comment:
I took the opportunity to refactor some options in unrelated tests that used the same options over and over again.